Fixed incorrect handshake header

Lolginer opened the pull request:
There were two mistakes in the header. The correct path is "recognition" rather than "recognize", and the ProtoSec-WebSocket-Key described does not work. Without these changes, any handshake results in 400 Bad Request.

Additionally, X-ConnectionId does not seem to matter. I don't know if that's an error or not, though.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Lolginer committed:
Fixed incorrect handshake header

There were two mistakes in the header. The correct path is "recognition" rather than "recognize", and the ProtoSec-WebSocket-Key described does not work. Without these changes, any handshake results in 400 Bad Request.

Additionally, X-ConnectionId does not seem to matter. I don't know if that's an error or not, though.

Lolginer committed:
Merge branch 'master' into patch-2

Lolginer committed:
Merge branch 'master' into patch-2
